tests/layer_tests/pytorch_tests/test_matmul.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+import pytest
+import torch
+from pytorch_layer_test_class import PytorchLayerTest
+
+class TestMVOperation(PytorchLayerTest):
+    def _prepare_input(self, matrix, vector):
+        matrix_input = np.array(matrix).astype(np.float32)
+        vector_input = np.array(vector).astype(np.float32)
+        return [matrix_input, vector_input]
+
+    def create_model(self, matrix, vector):
+        class CustomMVOperation(torch.nn.Module):
+            def forward(self, matrix, vector):
+                return matrix * vector  # Element-wise multiplication as per the provided code
```

Comment:
`# Element-wise multiplication as per the provided code`
I believe this should be a normal matrix x vector multiplication:
![image](https://github.com/openvinotoolkit/openvino/assets/58986814/58caaea2-df64-46df-bf33-8fc46c90bd39)
If it was indeed element-wise, the output would be a 3x3 matrix of ones.
```suggestion
                return torch.mv(matrix, vector)
```
Just to explicitly use the requested op.